snapcraft/tests/test_lxd.py

```diff
@@ -46,20 +47,27 @@ def test_cleanbuild(self, mock_pet):
 
         mock_pet.return_value = 'my-pet'
 
-        project_options = ProjectOptions()
+        with patch('platform.machine') as machine_mock, \
```

Comment:
I don't see how that's possible here. But I figured I can use decorators anyway so there's no need for the with at all.